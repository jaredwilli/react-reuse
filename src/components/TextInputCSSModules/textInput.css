/* TextInput using BEM styling principles */

.fieldset {
    margin-bottom: 16px;
}

.error {
    color: red;
}

.inputError {
    border: 1px solid red;
}
