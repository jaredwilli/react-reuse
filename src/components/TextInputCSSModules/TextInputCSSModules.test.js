import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import TextInputCSSModules from './TextInputCSSModules';

describe('TextInputCSSModules', () => {
    it('renders the component correctly', () => {
        const tree = renderer.create(<TextInputCSSModules />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
