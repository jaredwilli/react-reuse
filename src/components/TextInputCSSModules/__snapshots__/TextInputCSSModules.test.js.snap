// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInputCSSModules renders the component correctly 1`] = `
<div
  className={undefined}
>
  <label
    htmlFor={undefined}
    style={
      Object {
        "display": "block",
      }
    }
  >
     
  </label>
  <input
    className={undefined}
    id={undefined}
    name={undefined}
    onChange={undefined}
    placeholder={undefined}
    type="text"
    value={undefined}
  />
</div>
`;
